docs/using-flatpak.rst

```diff
@@ -8,24 +10,37 @@ commands.
 End users shouldn't generally need to use this page or the Flatpak command
 line interface, since Flatpak can be easily used through graphical software
 management tools, though they are of course free to use the command line if
-they prefer!
+they prefer! GNOME Software is one such example of a graphical application 
+to manage installing flatpaks.
```

Comment:
How about:

> since Flatpak can be easily used through graphical
management tools, such as GNOME Software, though they are of course free to use the command line if
they prefer!

Reply:
Swapped it out for your suggestion, that fits better.